column-samples/generic-right-align/generic-right-align.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/sp/column-formatting.schema.json",
+  "elmType": "div",
+  "style": {
+    "display": "table",
```

Comment:
Hi, came here with the same suggestion but came up with this method

`{
    "$schema": "https://developer.microsoft.com/json-schemas/sp/v2/column-formatting.schema.json",
    "elmType": "div",
    "children": [
        {
            "elmType": "span",
            "style": {
                "width": "100%",
                "text-align": "right",
                "margin-right": "20px"
            },
            "txtContent": "@currentField.displayValue"
        }
    ]
}`